apps/dashboard/app/helpers/batch_connect/sessions_helper.rb

```diff
@@ -195,6 +196,27 @@ def status_context(session)
     end
   end
 
+  def relaunch(status_array, session)
+    return unless Configuration.relaunch_session_enabled && session.completed?
+
+    batch_connect_app = session.app
+    return unless batch_connect_app.valid?
+
+    user_context = session.user_context
+    params = batch_connect_app.attributes.map{|attribute| ["batch_connect_session_context[#{attribute.id}]", user_context.fetch(attribute.id, '')]}.to_h
+    title = "#{t('dashboard.batch_connect_sessions_relaunch_title')} #{session.title} #{t('dashboard.batch_connect_sessions_word')}"
+    status_array << button_to(
+      batch_connect_session_contexts_path(token: batch_connect_app.token),
+      method: :post,
+      class: %w[btn px-1 py-0 btn-outline-dark relaunch],
+      form_class: %w[d-inline relaunch],
+      title: title,
+      data: { toggle: "tooltip", placement: "left" },
```

Comment:
```suggestion
      data: { toggle: "tooltip", placement: "left", selector: true },
```

For whatever reason, we need to add this flag here because jQuery/bootstrap tooltip does something funny with the title and it throws off screen readers.

You can see in the html `title` is empty, replaced by `data-original-title`. 
![image](https://user-images.githubusercontent.com/4874123/215798917-b3f31318-9313-4b16-b9ab-1e783279307d.png)

Reply:
Well one second, that actually breaks the hover-over tooltip.

Reply:
I noticed that while testing, the Delete tooltip was broken.
Reading about, the `selector` is needed when dynamic HTML is used and it should be a String with the locator of the tooltips elements. But I could not make it work locally.

In our case, we would need the selector for the card updates done with AJAX.

Reply:
> I noticed that while testing, the Delete tooltip was broken.

delete tooltip still works for me. Let's see if we can't get `aria-labeledby='data-original-title'` to work.

Reply:
This'll work for now, I'll create a follow up ticket to track tooltip buttons because I'd really like to use the title, but IDK if bs tooltips is going to let us.

```ruby
'aria-label': title,
```

Reply:
> delete tooltip still works for me

I see a tooltip, but it is the default HTML one, not the Bootstrap formatted:

<img width="250" alt="Screenshot 2023-01-31 at 16 09 57" src="https://user-images.githubusercontent.com/17336210/215816371-81434788-c43f-4f3a-aca8-61942b9e8ed0.png">

Is that what you see? or do you get the Bootstrap one?

Reply:
> Is that what you see? or do you get the Bootstrap one?

Oh OK, yes that is what I see, the default html.   I figure we can fix those by adding `'aria-label': title,` and removing selector as well.

Reply:
If you are happy with the default HTML tooltip, we can remove the whole:
 `toggle: "tooltip", placement: "bottom", selector: true`

This is only needed for Bootstrap tooltip.
By doing this, the default `title` will be available in the `button` element.

<img width="450" alt="Screenshot 2023-01-31 at 16 17 27" src="https://user-images.githubusercontent.com/17336210/215818079-5cc5836b-de0f-4e21-9693-b27796f43273.png">

Reply:
No, let's keep the tooltip because that does tell folks what this button does. I think just adding `'aria-label': title,` to that hash will give us both the tooltip for sighted users and the accessibility for screen readers.

```ruby
      # ...
      title: title,
      'aria-label': title,
      data: { toggle: "tooltip", placement: "left" },
      params: params
    ) do
```

Reply:
and we can fix the cancel/delete buttons in the same way.

Reply:
ok